### ISSUE TITLE
Manual paypal payments > $2k for 5 May 2018

### ISSUE THREAD

**Issue by alannallama** [opencollective/opencollective#1062]
Two payments need fees applied:

* https://opencollective.com/beaker/expenses/3472 - $2600 
* https://opencollective.com/webpack/expenses/3318 - $9500

**Comment by piamancini** [comment]
@alannallama what are the fees?

**Comment by alannallama** [comment]
@piamancini I don't know, how do I find out?

**Comment by piamancini** [comment]
before you hit send (the money) paypal displays the fees. I normally make a note then.  You can find them in the paypal dashboard, in activity, payments sent, search for the name of the recipient. 
let me know if you can't find them, happy to help

**Comment by alannallama** [comment]
@piamancini  I looked in Paypal and it says:

* https://opencollective.com/beaker/expenses/3472 - $2600 - fee $0.00
* https://opencollective.com/webpack/expenses/3318 - $9500 - fee $2.99 (payment pending)

Is that $0.00 correct? It says payment completed, no fee. ([link to transaction](https://www.paypal.com/activity/payment/1XK61080D1382515S))

**Comment by piamancini** [comment]
yes that makes sense, sometimes with paypal accounts in the same country there are no fees (the friends & family option). So the only one is 3318 (that looks super cheap for a txn of that size btw)
